apps/ledger-live-desktop/src/renderer/screens/exchange/index.jsx

```diff
@@ -61,6 +61,7 @@ const DEFAULT_MULTIBUY_APP_ID = "multibuy";
 const LiveAppExchange = ({ appId }: { appId: string }) => {
   const { state: urlParams } = useLocation();
   const locale = useSelector(languageSelector);
+  const activateNavigation = useFeature("activateNavigation");
```

Comment:
You might want a more specific feature flag for this, for example:
```suggestion
  const multibuyNavigation = useFeature("multibuyNavigation");
```